scanner.l

```diff
@@ -658,6 +658,7 @@ stou(const char *yytext_arg, YYSTYPE *yylval_arg, compiler_state_t *yyextra_arg)
 {
 	stoulen_ret ret;
 
+fprintf(stderr, "%s\n", yytext_arg);
```

Comment:
Should be removed from this PR.

Reply:
Yeah, that was me trying to figure out what that memory sanitizer issue was about.  Removed.

Reply:
I had the same...